### PR TITLE
Hey, check my long and hard work! :D

### PR DESCRIPTION
**Pull request by PrettyKawaii**
This my first pull request, so, I'll just show you my commits, enjoy! :)

### CODE REVIEW

File: Cars/Bus.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+namespace Cars
+{
+    class Bus:Car
+    {
+        public Bus(int seatingNum)
+        {
+            seatingNum = 40;
+        }
+        public static bool doorStatus;
```
Comment (gennadijbgl) [inline Cars/Bus.cs:15]: Hm, static... Only 1 bus will be affected. No more;)

File: Cars/Bus.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            {
+                case 1:
+                    doorOn();
+                    break;
+                case 2:
+                    doorOff();
+                    break;
+            }
+        }
+        public static void doorOn()
```
Comment (gennadijbgl) [inline Cars/Bus.cs:32]: Same here

File: Cars/Bus.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        {
+            doorStatus = true;
+            Console.WriteLine("Door was turned on!");
+        }
+        public static void doorOff()
```
Comment (gennadijbgl) [inline Cars/Bus.cs:37]: Same

File: Cars/Program.cs [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+            car.color = color;
+            car.kmTraveled = kmTraveled;
+            car.id = cars.Count;
+            cars.Add(car);
+            Console.WriteLine(" Car successfully added!");  
+        }
+
+        public static void outputCars()
+        {
+            //Console.WriteLine(" Output cars");
```
Comment (gennadijbgl) [inline Cars/Program.cs:75]: Clean this comment plz

**Review by gennadijbgl — CHANGES_REQUESTED** [review]
Very good, but y have to make field object dependency instead of static

File: Cars/Program.cs [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+        public static int manageKm()
+        {
+            Console.Write(" Enter, how much km you've want travel: ");
+            int travelKm = Int32.Parse(Console.ReadLine());
+            return travelKm;
+        }
+        public static void travel()
+        {
+            Car car = CarSystem.selectCar();
```
Comment (gennadijbgl) [inline Cars/Program.cs:107]: Y make a dependency of CarSystem. Get rid of it. We can discuss this, ill explain it

Comment (gennadijbgl) [reply Cars/Program.cs:107]: Here y can continue chat

Comment (PrettyKawaii) [reply Cars/Program.cs:107]: Like this?

Comment (gennadijbgl) [reply Cars/Program.cs:107]: yes

Comment (PrettyKawaii) [reply Cars/Program.cs:107]: Ok, soon I'll begin fix the bugs. :)

Comment (gennadijbgl) [reply Cars/Program.cs:107]: Perfect, actually it's not a bugs but some architecture advices =)

**Review by gennadijbgl — APPROVED** [review]
Decided refactor later

**Pull request merged**